.travis.yml

```diff
@@ -1,22 +1,31 @@
 os:
   - linux
-  - osx
 
 language: go
 
-addons:
-    artifacts:
-        debug: true
-        paths:
-        - $GOPATH/bin
-
 dist: trusty
 
 go:
-    - 1.6
+  - 1.7.4
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libpcap0.8-dev build-essential rpm
+  - gem install fpm
 
-script: go test github.com/honeycombio/rdslogs/...
+script:
+  - go test github.com/honeycombio/rdslogs/... -bench=.
+  - go vet ./...
 
 after_success:
-    - rm $GOPATH/bin/rdslogs
-    - go install -ldflags "-X main.BuildID=1.${TRAVIS_BUILD_NUMBER}" github.com/honeycombio/rdslogs/...
+  - rm $GOPATH/bin/rdslogs
+  - go install -ldflags "-X main.BuildID=1.${TRAVIS_BUILD_NUMBER} -extldflags \"-static\"" github.com/honeycombio/rdslogs/...
```

Comment:
`-extldflags \"-static\"` also not needed here (honeycomb-tcpagent only needs that to statically link libpcap)